### PR TITLE
Fixes mm-1420 adding initial support for postgres

### PR DESCRIPTION
**Pull request by coreyhulen**
has config file changes.

### CODE REVIEW

File: store/sql_channel_store.go [hunk 1]
```diff
@@ -167,7 +169,7 @@ func (s SqlChannelStore) Delete(channelId string, time int64) StoreChannel {
 	go func() {
 		result := StoreResult{}
 
-		_, err := s.GetMaster().Exec("Update Channels SET DeleteAt = ?, UpdateAt = ? WHERE Id = ?", time, time, channelId)
+		_, err := s.GetMaster().Exec("Update Channels SET DeleteAt = "+s.GetMaster().Dialect.BindVar(0)+", UpdateAt = "+s.GetMaster().Dialect.BindVar(1)+" WHERE Id = "+s.GetMaster().Dialect.BindVar(2), time, time, channelId)
```
Comment (coreyhulen) [inline store/sql_channel_store.go]: Check to make sure we cannot use named params

Comment (crspeller) [inline store/sql_channel_store.go]: What was special about this query that required this form over the mapp[string]interface{} form?

File: store/sql_store.go [hunk 2]
```diff
@@ -127,6 +123,11 @@ func setupConnection(con_type string, driver string, dataSource string, maxIdle
 }
 
 func (ss SqlStore) DoesColumnExist(tableName string, columnName string) bool {
+	// XXX TODO FIXME this should be removed after 0.6.0
```
Comment (crspeller) [inline store/sql_store.go]: This refers to the entire function and will be removed when we implement database schema versions. Correct?

File: store/sql_channel_store.go (hunk 1 above)
Comment (coreyhulen) [inline store/sql_channel_store.go]: nope.  good catch.  I thought I fixed that.

File: store/sql_store.go (hunk 2 above)
Comment (coreyhulen) [inline store/sql_store.go]: correct

Comment (coreyhulen) [inline store/sql_store.go]: I've also added a comment to MM-1698 to remove the alter table methods but leave the index creation method.

**Comment by jwilander** [comment]
Looks good to me

**Comment by crspeller** [comment]
Looked at the changes. Looks good to me as well.

**Pull request merged**